src/systems/filecoin_mining/storage_mining/_index.md

```diff
@@ -19,36 +18,44 @@ The Filecoin Storage Mining Subsystem ensures a storage miner can effectively co
 
+A particular sector enters `TemporaryFault` from `Active` through `DeclareTemporaryFault` with a specified period. Power associated with the sector will be lost immediately and miner needs to pay a `TemporaryFaultFee` determined by the power suspended and the duration of suspension. At the end of the declared duration, faulted sectors automatically regain power and enter `Active`. Miners are expected to prove over this recovered sector. Failure to do so may result in failing ElectionPoSt or `DetectedFault` from failing SurprisePoSt. 
 
-A sector that is in the `Failing` state for three consecutive `Proving Period`s will be terminated (`Terminated Faults`) meaning its data will be deemed unrecoverable by PoSt and the sector will have to be SEALed once more (as part of PoRep).
+{{< diagram src="diagrams/sector_state_machine.dot.svg" title="Sector State Machine" >}}
 
-Conversely `RecoverFaults()` can be called any time by the miner on a failing sector to return it to the `ProvingSet` and attempt to prove data is being stored once more. For instance an Active sector might move into the failing state during a power outage (through a declared or detected fault). At the end of the outage, the miner may call `RecoverFaults` to transition the state to `Recovering` before proving it once more and returning it to `Active`. 
+{{< diagram src="diagrams/sector_state_machine_legend.dot.svg" title="Sector State Machine Legend" >}}
 
-We illustrate these states below.
+### Miner PoSt State
 
-{{< readfile file="storage_mining_subsystem.id" code="true" lang="go" >}}
+`MinerPoStState` keeps track of a miner's state in responding to PoSt and there are three states in `MinerPoStState`:
 
-### Sector in StorageMiner State Machine (new one)
+- `OK` miner has passed either a ElectionPoSt or a SurprisePoSt sufficiently recently.
+- `Challenged` miner has been selected to prove its storage via SurprisePoSt and is currently in the Challenged state
+- `DetectedFault` miner has failed at least one SurprisePoSt, indicating that all claimed storage may not be proven. Miner has lost power on its sector and recovery can only proceed by a successful response to a subsequent SurprisePoSt challenge, up until the limit of number of consecutive failures.
 
-{{< diagram src="diagrams/sector_state_fsm.dot.svg" title="Sector State (new one)" >}}
+`DetectedFault` is a miner-wide PoSt state when all sectors are considered inactive. All power is lost immediately and pledge collateral is slashed. If a miner remains in `DetectedFault` for more than MaxConsecutiveFailures, all sectors will be terminated, both power and market actors will be notified.
```

Comment:
If power was lost when challenged (not documented, but I heard it verbally), it's not lost (again) when the challenge expires un-met.